packit_service/worker/handlers/testing_farm_handlers.py

```diff
@@ -84,12 +84,16 @@ def run(self) -> TaskResults:
 
         if self.result == TestingFarmResult.running:
             status = CommitStatus.running
+            summary = self.summary or "Tests are running ..."
```

Comment:
Will they send the summary? Is it some planned feature or just a fallback?

Reply:
@thrix said yesterday that there should be something like "1 test passed, 1 failed" in the summary, not sure what's the current status

Reply:
yeah, we will, sorry still not done, but I am close ...